### ISSUE TITLE
Chapter 1 feedback

### ISSUE THREAD

**Issue by Talon1024** [Realm667/WolfenDoom#155]
Please enclose the screenshots you post into spoilers, so the issue does not become a monstrosity like #178.
---

<details><summary>Original post by Talon1024</summary>

So, I decided to do a run of chapter 1 today with [this commit](https://github.com/Realm667/WolfenDoom/commit/c54714892c06df6597fa8c209291a702e29c48ee) and here are my thoughts:

C1M2: Allowing the mutants to throw their cleavers certainly mixes things up a bit.
C1M3: Good idea moving one of the Comp B charges to a different spot. That definitely got me through the mission faster. Also, maybe the Activate script can be added to when you set the bombs on the artillery cannons.
C1M4: Again, maybe add the Activate script to the flak guns.
C1M5: It's not finished yet. I thought of adding the Activate script to the bombs on the bridge (with a very short time). I also noticed the Allied reinforcements don't show up any more. Also, I didn't find the machinegun very helpful, and it seems to have made me more of a sitting duck than I would hope. Speaking of which, I'd appreciate a manual reload function, as well as a heat gauge on the HUD (it doesn't have to be anything fancy, just display the ammo in the primary ammo slot on the HUD, and display the heat in the secondary ammo slot)
C1M6: The player's ammo is rather limited at the start. Can we have some more bullets and shells? The red gas cloud left behind by the skulls looked like it could hurt me like the mutants' poison gas clouds. I also noticed lava doesn't cause instant death any more. Speaking of the lava, though, once I pushed the brick to raise the blocks out of the lava, I got a MASSIVE performance drop.
C1M0: The mutants that launch gas missiles certainly seem to be less frustrating than they were before. However, when I go to pick up the key beside crate 2, I notice the blockade does not close behind me. I made it work that way for Chapter 2 so that players wouldn't get stuck in the mission after destroying the facility and forgetting to bomb a crate or collect a key.

Also, I noticed that BJ's standing sprites on TITLEMAP and the C1 outro conflict with the SS occult fire FX.

</details>

<details><summary>Resolved issues</summary>

**General:**

- [x] The tracer pitch issue returns. @AFADoomer?
> It has been fixed on this commit by AFA https://github.com/Realm667/WolfenDoom/commit/1d22c6b610e94071645afcaca273044d0cc035a4 --Ozy81

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200715_225144](https://user-images.githubusercontent.com/59409713/87589391-d5847d00-c6ed-11ea-8d67-83dce915cc18.png)

</details>

**C1 INTERMAP:**

- [x] Training targets in INTERMAP have infinite height.
> They had a completely flat behavior, [added](https://github.com/Realm667/WolfenDoom/commit/380dfca4837daa4c9ea6ad892a34be210422d423) something that now makes them very cool. --Ozy81

- [x] After returning to INTERMAP from a mission the statistics' lines stay lowered.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200925_104335](https://user-images.githubusercontent.com/59409713/94364708-12002c00-00d4-11eb-8215-f62c7fb7b187.png)

</details>

**C1M1:**

**C1M2:**

- [x] Got 253 of 254 kills and 39 of 41 items, could not find the remaining ones with iddt x2.

> This needs retesting. --N00b

> False alarm on items (here are 45 now), 242 of 243 kills. StatistDarren has +COUNTKILL. --N00b

- [x] Large invisible box over the loudspeakers in the Dr. Schabbs Jr. mutants' room (you can leave decals on it). Also a portal inconsistency there (as you can walk into Schabbs' room after killing him).

**C1M3:**

**C1M4:**

- [x] Initial scene is too non-realistic: Germans face player and Ryan but do not see them and do not react; Ryan does not turn.

- [x] Camera flinches in lifts a bit (no more than 2–3 units).

- [x] Two or so additional enemies on killcount, see also C1M0.

- [x] 150mm rockets stacked near 20mm Flakvierlings... :-|

- [x] If you jump off the dam in C1M4 to the water's falling direction, you can land on a 3d floor, the only escape from which is to grenade-jump back. The 3d floor needs to be raised 16 units up.

- [x] The player can shoot Ryan off the ramp in C1M4 when he walks on it.

- [x] Managed to get this soldier not killed by Ryan somehow:

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_211214](https://user-images.githubusercontent.com/59409713/94364625-753d8e80-00d3-11eb-92ce-872c024ad535.png)

</details>

- [x] Stabbing Ryan here shows this sprite. **Please block NPCs with Impassable + Block Projectiles + Block Hitscans flag combination or similar ones. This is also a problem of other maps, including C2M4 which is shown in my videos.**
> Simply setting Ryan Invulnerable and nonshootable does the trick, this via scripts --Ozy81

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_211248](https://user-images.githubusercontent.com/59409713/94364636-8a1a2200-00d3-11eb-9f13-21283800d5d6.png)

</details>

- [x] In this photo I am immediately facing a linedef which is blocking for no reason.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_211436](https://user-images.githubusercontent.com/59409713/94364641-9e5e1f00-00d3-11eb-9f84-d620cc5caf93.png)

</details>

- [x] Portals gonna portal. Can this one be changed to static or, even better, removed?
> I still recall that portal and it is a rare situation where you have the "traverse" issue not solvable unless you add another portal on the nearest zone. I did [this](https://ibb.co/z8dJQTZ) but there is no solution to avoid the upper portal being impassable by players due to the fact that heights are different between lines, I even flipped linedefs with portals specials, or crossed them but there isn't a feasible solution and we can't even use Eternity portal though. So the zone will be kept as is, unless @Tormentor667 or you @Username-N00b-is-not-available manage to find a different solution.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_211952](https://user-images.githubusercontent.com/59409713/94364653-b766d000-00d3-11eb-9aaa-4641b2158f5e.png)

</details>

- [x] The player does not have to kill these soldiers to bomb the penultimate AAA piece at all.

> Well doing that will not allow players to end the map 100%, so it is up to players choice, here it is a different situation compared to the Tank fight at the end of C3M1_A --Ozy81

> I think this is not :x:, please see the screenshot. I meant that Ryan and Douglas spawn right behind the enemies and do not fight with them. Sorry for my very poor wording, wanted to tell you about the ending of the map, but wrote this instead. --N00b

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_214425](https://user-images.githubusercontent.com/59409713/94364657-c9e10980-00d3-11eb-8989-47303e011a31.png)

</details>

**C1M5:**

- [x] Player can get temporarily stuck here (thankfully they are teleported to another location):

> This problem persists. Maybe it would be better to make an interactive sector portal here. --N00b

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200202_150309](https://user-images.githubusercontent.com/59409713/73612024-19c90880-45f9-11ea-8368-a8058058731a.png)

</details>

- [x] Somehow got a second chest key (see savegame).

- [x] 59 of 95 items with full secrets; tried 'remove SingleCoin', it writes that zero are found... I did visit the bank. See also next checkbox.
> Checking through UDBuilder made me notice that there are 46 treasure related actors...
![image](https://user-images.githubusercontent.com/11136108/89644268-d3aa7580-d8b7-11ea-9333-20ad280bac16.png)

> ...plus the treasure chest which drops 46 items (100coins total) - 6 bagofcoins = 60 coins and 40 singlecoin. So there are still 2 left (they are 94 treasure at map start now 07/08/2020), but need to play the map to be sure --Ozy81

> This was a false positive, sorry --N00b

- [x] Managed to get a zero-hp WaffenSS who is walking and shooting, and cannot be killed once more. Savegame (g4.4-15, yesterday pk3 named as wolf_boa-latest.pk3): [save90.zip](https://github.com/Realm667/WolfenDoom/files/4144794/save90.zip)

> This is lost in the sands of time... --N00b

> The bug persists (15 Jul 2020), this time with a Wehrmacht officer --N00b

- [x] Super-soldiers get 'stuck' behind scenery tanks in wave 7 of C1M5's final battle, constantly firing at the player if they are in their LOS. This becomes even more noticeable on Mein Leben.

- [x] Knifing Juliette and resistance fighters, who can't even retaliate, is very cruel... but why wouldn't I, I'm a tester. **Again, please surround them with blocking lines, then remove the blocking flags after their scripted departure.**

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_215123](https://user-images.githubusercontent.com/59409713/94364684-ea10c880-00d3-11eb-8233-f9b99a220785.png)

</details>

- [x] The mutants can see and shoot through this door.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_221026](https://user-images.githubusercontent.com/59409713/94364700-0876c400-00d4-11eb-85ae-f908c58dede0.png)

</details>

**C1M0:**

- [x] Green door area should be locked when the player has not obtained the next key.

- [x] Many (if not all) railings in the complex block projectiles.

> Mostly fixed, only the 3dfloor railings in the central area remain, but it would be difficult to change them to walkable linedefs, and the effect is not worth it. --N00b

- [x] 260 of 252 kills (to retest later).

>Retested & confirmed, exactly 260! --N00b, 15 Jul 2020

> 260 of 260 now --N00b, 22 Oct 2020

- [x] 3d tree leaves clip into building.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200925_113724](https://user-images.githubusercontent.com/59409713/94364756-5db2d580-00d4-11eb-9497-0985d323f338.png)

</details>

- [x] Liquid sectors in this area have wrong and inconsistent displacement vectors (best seen in action).

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200925_173845](https://user-images.githubusercontent.com/59409713/94364762-6a372e00-00d4-11eb-9408-45cc5cba1a67.png)

</details>

**C1M6:**

- [x] On Mein Leben the scripted fight between cultists and guards does not work as intended:

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200823_162019](https://user-images.githubusercontent.com/59409713/91454890-1ee9ef80-e88a-11ea-9215-a22b16340db5.png)

</details>

- [x] Possibly wrong flat and armor sprite clipping into sector:

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_194940](https://user-images.githubusercontent.com/59409713/94177089-b5b5c600-fea1-11ea-9bfc-077a8493d6ec.png)

</details>

- [x] These soldiers can be moved by grenades and other AoE damage (see also ending of C2M0_A and possibly other maps), also caught a blue pixel in the flamer's smoke sprite:

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200924_200209](https://user-images.githubusercontent.com/59409713/94177123-c1a18800-fea1-11ea-872f-eaa4baccf985.png)

</details>

- [x] Move the script activation from linedefs to item pickup specials? (video) [c1m6_1.zip](https://github.com/Realm667/WolfenDoom/files/5277914/c1m6_1.zip)

- [x] You can jump down here, and it will be possible for you to escape only by using a damage boost.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200925_105853](https://user-images.githubusercontent.com/59409713/94364726-27755600-00d4-11eb-9187-235890b91dbb.png)

</details>

- [x] Improper texture alignment? Possibly a false positive.

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200925_110753](https://user-images.githubusercontent.com/59409713/94364733-3825cc00-00d4-11eb-8422-a978f717da3e.png)

</details>

- [x] On Mein Leben the super-soldier continuously flings grenades at the player, but the distance is too large for the grenades to reach the player location, so he becomes a sitting duck. _[assigned to @Ozymandias81]_

- [x] The boulders sometimes stop on the boundary here. _[assigned to @AFADoomer]_

<details><summary>Screenshot</summary>

![Screenshot_Doom_20200925_105822](https://user-images.githubusercontent.com/59409713/94364715-162c4980-00d4-11eb-84ae-d935258f6466.png)

</details>

</details>

![tumbleweed](https://thumbs.gfycat.com/ForthrightLonelyIberianbarbel-size_restricted.gif)

**Comment by Ozymandias81** [comment]
**> C1M2: Allowing the mutants to throw their cleavers certainly mixes things up a bit.**
I remember we wanted to have battles with different enemy behavior, sometimes even unpredictable, so thought the feature proposed by DoomJedi was fine.

**> C1M5: I also noticed the Allied reinforcements don't show up any more.**
This [should have](https://github.com/Realm667/WolfenDoom/commit/bce6d4b2248b4400b1177dc6e5228414d1aa8319#diff-3a7061233c12e1d5ed41c4b9a149136e) [been fixed](https://github.com/Realm667/WolfenDoom/commit/d55958ddfb9bd245668379d41fd4a4ed11f0800f#diff-3a7061233c12e1d5ed41c4b9a149136e) before, maybe some new tags have been added and cause conflicts.

**> C1M6: The red gas cloud left behind by the skulls looked like it could hurt me like the mutants' poison gas clouds**
I added that effect to reproduce some kind of blood haze when they die, and I think the fact that you could be foolished by it would be nice in game causing some panicking - only the first time you see it

**> Also, I noticed that BJ's standing sprites on TITLEMAP and the C1 outro conflict with the SS occult fire FX.**
How so if I have renamed sprites for them [several commits ago](https://github.com/Realm667/WolfenDoom/commit/31c24b14c03c0540eb92c1522395c0b3bfb56c18#diff-1299c668d881b429e5bd27e239f68a73)? Now BJ cutscene frames are BLAZ, but Flame attacks use BLZF name

**Comment by AFADoomer** [comment]
C1M6 lag issue should be resolved now.  I think the engine was basically doing all of the collision checks as if the platforms were trying to crush the (invisible) HeatEffectGiver actors against the ceiling, and, since those actors don't take damage, causing huge lag as long as the platforms were moving.

**Comment by Talon1024** [comment]
> How so if I have renamed sprites for them several commits ago? Now BJ cutscene frames are BLAZ, but Flame attacks use BLZF name
I use incremental builds of WolfenDoom, built with my own personal shell scripts. I load up a base package, and then add the new stuff on top of it, as well as files which, if changed, could break the game (those being zscript.txt, modeldef.txt, decorate.txt, and cvarinfo.txt).

**Comment by Ozymandias81** [comment]
> **I use incremental builds of WolfenDoom, built with my own personal shell scripts. I load up a base package, and then add the new stuff on top of it, as well as files which, if changed, could break the game (those being zscript.txt, modeldef.txt, decorate.txt, and cvarinfo.txt). />

Any different solution you could adopt then? I suggest to build pk3 from the last commit.

> **Also, I have no idea why, but INTERMAP crashes on the first lesson of the training course when the bullet you fire hits the target.** />

I fear we should wait some newer builds, could be a "projectile crosses" bug with scripts activation (script 801 in that case

Anyone can confirm that recent builds of GZDoom have IPK3 support broken? Can't run BoA with recent builds since r345 (gonna try previous)

**Comment by Ozymandias81** [comment]
[This commit](https://github.com/Realm667/WolfenDoom/commit/ea9d22ad5210a36f13d0c1f07052528a28bbd9ea) should fix C1M5 battle at least, we can control the allied support arrive now with script 301 (set for wave 5). Please test in game properly.

**Comment by AFADoomer** [comment]
INTERMAP training course crash fixed.  This was an infinite loop that would occur in certain cases in the collision check function of the BulletTracer class.

**Comment by Talon1024** [comment]
> INTERMAP training course crash fixed. This was an infinite loop that would occur in certain cases in the collision check function of the BulletTracer class.

Not for me. I did some further investigation with a debug build of GZDoom 3.3.2, and found that the tanks on the training course also caused crashes.

Commenting out [line 367](https://github.com/Realm667/WolfenDoom/tree/master/scripts/actors/enemies/tanks.txt#L367) of scripts/actors/enemies/tanks.txt, as well as setting the "Stand still" flag on the tanks fixed the crash for me. This crash may even be a bug with A_LookEx.

**Comment by AFADoomer** [comment]
I don't get it.  I just re-ran my local version and now I get the crash again.  And I *know* that it wasn't crashing before.

**Comment by AFADoomer** [comment]
OK, just pushed a new commit that doesn't actually fix the bug, but fixes the INTERMAP tanks so that they don't cause the crash anymore (I added a 'user_static' variable to the tanks so we can set up proper destroyable static tanks that don't play an idle sound).

**Comment by Tormentor667** [comment]
@Talon1024 
> C1M2: Allowing the mutants to throw their cleavers certainly mixes things up a bit.
If this hasn't been added: I am all okay for it
> C1M3-C1M4-C1M5: Activate scripts
I am all okay for adding these, would you volunteer doing it?
> C1M5: Machinegun ammo and stuff
Also okay with the suggestion, but I think this is something @AFADoomer needs to take care, no idea how to code that.
> C1M6: More ammo
Also okay, if you can do that, I'd appreciate it.
> C1M0: Buggy bugs
Same here, would be happy if you can take care.

**Comment by Ozymandias81** [comment]
@AFADoomer and @Talon1024 I think the best and easy way to fix the issue on INTERMAP with Tanks (yet the enemy counter is negative because of them) is to make a separate variant which doesn't inherit with Tanks but just act as a placeholder actor that can show the healthbar and be killed exactly like normal tanks.
Will try something later.

**Comment by AFADoomer** [comment]
I can probably just make tanks with user_static set remove the COUNTKILL flag.  That should fix the kill counter.  Plus there is probably no need for the tanks to be set as FRIENDLY any more, either, come to think of it.

**Comment by Ozymandias81** [comment]
Leaving here feedbacks from the russian dude which is helping use on translations, gonna give it a go through fixes (I leave them here even if these contains some stuff which doesn't belong to C1):

In C1M1 a player can use sprint+jump to get from the first rooftop 'money' secret to the second, even with unlimited sprint off, and then jump to the area with the key. I think this, while clearly unintended, is quite a remarkable speedrunning feature of the map, and do not request to change it; I would rather ask to fix the back of the extinguisher-destroyable rock wall — it just looks out of place. By the way, are the rocks near the extinguisher filled with some high-grade explosives? How could its explosion make such a big hole in the brick wall AND in the rocks (and as for the latter, the explosion wave somehow only proceeds in one direction)??!

In C1M3, the teleporting lift on (3740, 1280, 24) produces a visual glitch when the player looks up, as the shaft texturing (and structure!) are not identical in teleport areas. I suggest either building a copy of the upper floor over the lower or just to put a visual plane portal there :-)
Apparently, mission summaries (which are viewed in the briefing room) do NOT get written into savegames.

In C1M4, there is some way to get 133 of 131 kills... (at least on ITYTD).

In C1M5, an instance of J. Bertrand who gets teleported to the hotel hall in the end of the mission can be killed; she has 1000hp like a decoration, but can be damaged; on her death, the actor is removed and the special (message) is executed.

C1M0: on (4448, 1665, 192) the player can fall between the sector rock where the first sniper stands and a 3D-model rock, and there is no escape except idclip (or multi-grenade jump, but this ought to be fatal :-) ).
From (12866, 935, 232) the player can sight a dormant Pzkpfw. IV, and it could be reached with a Panzerschreck rocket from there — with zero effect, of course. Wasting a rocket in a such manner would be excessively frustrating, especially for a newbie.

Some common suggestions:
1. 2D trees surely are a placeholder for low-end hardware, but will there be any adequate textures for them? Sprites, maybe? On maps such as the one with usable Sherman (C2M4?) there are some scaled trees which look just plain ugly.

2. Why can the tanks be damaged by everything starting from pistol bullets & shotgun pellets and ending with quick kicks? I can unload my shotgun or an SMG into it, and it will explode after just a few magazines. Is that intentional? As for Pzkpfw. IV, less than 250 bullets are needed for this (it can be checked with an UMG43, it goes down in seconds). Its death is even faster with the flamethrower's primary fire. Isn't their armor supposed to protect the crew from such bullet attacks? Also on tanks:
The SdKfz. 251 is an APC, not a tank (why does Douglas name it a tank?), yet it has a whopping 4000 HP; Pzkpfw. IV has only 3500, while it is a medium tank; and Pzkpfw. VI Tiger, which is a heavy tank with very strong armor, has only 3000! SdKfz. 222 is also wrongly labelled as 'Panzer KFZ 222'.
When any tank gets destroyed, its turret gets reverted into the original position (are the destroyed tank models just not split yet?)
I suggest balancing the HP of armored vehicles (e.g. change SdKfz. 251's to 1500 and Tiger's to 6000, leaving Pzkpfw. IV as is, so that they could be destroyed on UV using 1, 3 and 2 Panzerschreck rockets respectively) and adding a DamageFactor 0.5 for rifle bullets, 0.0 for pistol and shotgun projectiles and melee attacks (maybe except Electroschocker) and 0.25 for the flamethrower.

3. Are the Astro-Luger sprites a placeholder? They do not fit well. Why is Tesla Cannon so weak (against human opponents, at least)? Also, would it be a good idea to give the secret weapons the inventory flag +WEAPON.CHEATNOTWEAPON so that they could not be accessed by idfa?

4. Enemy bullet tracers pierce through models (and doors in toilets, which seem to be 3D floors!), while the actual bullets don't. Movable Nazi crates which are unaligned to the world axes have improper collision boxes, they intersect with walls when bumped into them.

5. In some maps (e.g. Fuehrerbunker with Rastenburg — C3M4?) there are noticeable 'angles' between flat and sloped sectors. Will they get smoothed closer to the release?

**Comment by Tormentor667** [comment]
Regarding killcounts and itemcounts, I wonder, do the statistics take it into account, if additional enemies or items are spawned through acs during the map @AFADoomer @Talon1024 

By the way, I am pretty sure that 100% items isn’t possible in C1M6. You can’t get out of the treasure room alive if you collect it all.

**Comment by AFADoomer** [comment]
The kill and item counts are the regular GZDoom code, so yes.

Most of the problems I've seen with counts are with Friendly enemies who are already +FRIENDLY in Decorate or ZScript being tagged Friendly in the map (causes them to be counted wrong).  

I thought these count issues were mostly fixed?

**Comment by Tormentor667** [comment]
I am going to do a full playtest of Chapter 1 in the next few days with getting a 100% ratio on everything. I will report back on this on a different issue then to keep track of the things in a more well-arranged way as I find this thread very hard to follow.

**Comment by Tormentor667** [comment]
@Username-N00b-is-not-available - Can you share screenshots with coordinates for the following issues:

- C1M4: 150mm rockets stacked near 20mm Flakvierlings
- C1M0: Many (if not all) railings in the complex block projectiles.
- C1M0: Green door area should be locked when the player has not obtained the next key.

**Comment by Ozymandias81** [comment]
Some stuff present on this issue may be redundant or already fixed, mind you. Fact is most of these requires proper playthrough so I have kept for later.

**Comment by Username-N00b-is-not-available** [comment]
@Tormentor667: The up-to-date information is in the first post. Everything else is probably obsolete, having been fixed months ago.

Sorry, I have limited internet connection, so no screenshots for now; as for the coordinates:

- C1M4: 150mm rockets stacked near 20mm Flakvierlings

Coordinates: (-1700, y, -8), where y varies from 1300 to 7400. The sixth AA gun does not have these.

- C1M0: Many (if not all) railings in the complex block projectiles.

The central elevator ones (12921, -3157, z), z from -1100 to 0; (17310, -2686, -224) (crouch down, and you won't even be seen by enemies). I'd use the 'Invert Shootability' flag on them (damn, now I realize I should have done it on all the lifts I edited...)

- C1M0: Green door area should be locked when the player has not obtained the next (red) key.

The doors near (13594, -3035, -352). This was more of a suggestion than an issue, and it also applies to the red/blue keys.

**Comment by Tormentor667** [comment]
Thanks, going to fix now.

**Comment by Username-N00b-is-not-available** [comment]
Well, IIRC when I added this issue all the areas with other keys locked the player in so they would not lose time searching for the next key in previously-visited areas. This behaviour might have been removed, I'm not sure about the reasons.

**Comment by Tormentor667** [comment]
Fixed both remaining C1M4 bugs in my next commit

**Comment by Ozymandias81** [comment]
This issue, unless some else pops up, could be closed finally. With my last commit I have fixed remaining stuff.

**Comment by Username-N00b-is-not-available** [comment]
>unless

*until.

**Comment by Tormentor667** [comment]
This feels so good :)

**Comment by Alexander-Wilms** [comment]
I noticed in the C1 Intermap that soldiers can get stuck in a doorway when trying to open the door at the same time as they do. I'm using GZDoom 4.5.0 and [this](https://github.com/Realm667/WolfenDoom/commit/e875fa16001858515ce55862596925b439cddb5f) Wolfenstein commit

<details>

![Screenshot_BoA_20210124_150950](https://user-images.githubusercontent.com/3226457/105633176-6c798100-5e57-11eb-8178-030f585d421b.png)
</details>

[Savegame](https://www.dropbox.com/s/bhran7othdc1bh0/save1.zds?dl=0)
